lib/utils.js

```diff
@@ -647,7 +647,7 @@ function maybePromise(callback, wrapper) {
   wrapper(function(err, res) {
     if (err != null) {
       try {
-        callback(err);
+        callback(err, null);
```

Comment:
this was giving me trouble in a test where it expected result to be null and NOT undefined